stories/core/Table.stories.tsx

```diff
@@ -1335,12 +1336,16 @@ export const Full: Story<Partial<TableProps>> = (args) => {
         rowProps={rowProps}
         isSelectable
         isSortable
+        isResizable
         {...args}
       />
       <Tooltip
         reference={rowRefMap.current[hoveredRowIndex]}
         content={`Hovered over ${data[hoveredRowIndex].name}.`}
         placement='bottom'
+        followCursor
```

Comment:
Slipped in story update from the discussion.

Reply:
So are you going to undo it? 😄

Reply:
Too much work, I'll just leave it here and hope nobody sees it 😁

Reply:
at least add `initial`. It is considered outdated UX practice for tooltip to follow cursor.

Reply:
Do you have a link for this guideline?

Reply:
Check our standards.

The guideline was updated in BWC standards first:
> The tooltip should be anchored to the item that is being hovered and should not follow the cursor as it moves.

And then refined in iTwinUI standards:
> It is anchored to the hovered item and fades out when the cursor moves away from the item.

Reply:
Removed for now.